### PR TITLE
Add: Yung Ape Dao

### PR DESCRIPTION
**Pull request by thedeini**

### CODE REVIEW

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]

File: protocols/yungapedao/index.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "Yung Ape Dao",
```
[the remark this answers is not in the document]
Comment (thedeini) [reply protocols/yungapedao/index.json]: Wow, I messed up big time on my second commit but fixed it right after, can you check and verify if everything is alright? Or should I open another PR? Sorry for this problem...

File: protocols/yungapedao/index.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "YungApeDao",
```
Comment (Rashmi-278) [inline protocols/yungapedao/index.json:2]: please change all the letters to small , this is an parameter to retrieve protocols and will not displayed.

Comment (Rashmi-278) [inline protocols/yungapedao/index.json:2]: please make it all smallcase, this parameter is not used for display, we use it to retrieve protocols

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]

File: protocols/yungapedao/index.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rashmi-278) [reply protocols/yungapedao/index.json]: No worries, we can keep this PR :)

[the remark this answers is not in the document]
Comment (thedeini) [reply protocols/yungapedao/index.json]: Thanks for that :)

**Review by Rashmi-278 — DISMISSED** [review]

**Comment by Rashmi-278** [comment]
Closing this as new PR will be added focusing only on the relevant changes to files with logo addition.

**Pull request closed without merging**